### DISCUSSION TITLE
How to get real-time data values from form?

### DISCUSSION THREAD

**Discussion by vitali-duckma** [ant-design/ant-design#40229] [Q&A]
https://ant.design/components/form#components-form-demo-usewatch
I'd need something like this but with multiple inputs, pls help. thanks

**Comment by zombieJ** [comment]
Just give empty path: https://codesandbox.io/s/watch-hooks-antd-5-1-4-forked-p00c1p

**Reply by vitali-duckma** [reply]
Ok thanks it works, but how to make it undefined after clear input?
![Screenshot 2023-01-13 alle 17 01 07](https://user-images.githubusercontent.com/118166188/212364537-ccb9458e-7cda-4bb0-ac3b-a1d8e51262b0.png)

**Reply by vitali-duckma** [reply]
@zombieJ seems a bug, because if you delete form item value, it maintans the field name, but need to destroy the form object
![Screenshot 2023-01-13 alle 17 10 00](https://user-images.githubusercontent.com/118166188/212366413-5b5313a9-8b70-496b-9c97-612f6e316832.png)

**Reply by zombieJ** [reply]
Delete by input is just mean leave empty string. Use `form.resetFields()` is real clean up.

**Reply by vitali-duckma** [reply]
how to create that function? and where to call it, because i don't want additional buttons to reset field?? thanks
